get_all_manifests.sh

```diff
@@ -25,10 +25,21 @@ MANIFEST_RELEASE="master"
 MANIFESTS_TARBALL_URL="${GITHUB_URL}/${MANIFEST_ORG}/odh-manifests/tarball/${MANIFEST_RELEASE}"
 mkdir -p ./.odh-manifests-tmp/ ./odh-manifests/
 wget -q -c ${MANIFESTS_TARBALL_URL} -O - | tar -zxv -C ./.odh-manifests-tmp/ --strip-components 1 > /dev/null
-cp -r ./.odh-manifests-tmp/model-mesh/ ./odh-manifests
-cp -r ./.odh-manifests-tmp/odh-model-controller/ ./odh-manifests
-cp -r ./.odh-manifests-tmp/modelmesh-monitoring/ ./odh-manifests
-cp -r ./.odh-manifests-tmp/kserve/ ./odh-manifests
+
+# ODH manifests
+ODH_MANIFESTS=(
+    "model-mesh"
+    "odh-model-controller"
+    "modelmesh-monitoring"
+    "kserve"
+    "trustyai-service-operator"
+)
+
```

Comment:
i am fine with this change, but i think trusty need to work on migration just like other components.
these left-over components will get the work done in the next week or so,
the goal is to empty these lines before 1.35 feature freeze done for downstream, i.e for upstream we will have it as soon as possible to verify all working before moving to downstream @ruivieira

Reply:
@zdtsw I'll rebase on https://github.com/opendatahub-io/opendatahub-operator/pull/585 and target `incubation` 👍